moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectECapris.js

```diff
@@ -158,18 +156,12 @@ const ProjectSummaryProjectECapris = ({
               onChange={handleProjectECaprisChange}
               value={eCapris}
             />
-            <Icon
-              className={classes.editIconConfirm}
-              onClick={handleProjectECaprisSave}
-            >
-              check
-            </Icon>
-            <Icon
-              className={classes.editIconConfirm}
-              onClick={handleProjectECaprisClose}
-            >
-              close
-            </Icon>
+            <ProjectSummaryIconButtons
+              handleSave={handleProjectECaprisSave}
+              handleClose={handleProjectECaprisClose}
+              disabledCondition={originalValue === eCapris}
```

Comment:
it is possible to add the `isValidECaprisId` condition here so you can't save invalid IDs but then you wouldn't get the error message that tells you the proper format. it would probably be better to have an inline error message with a red box and keep this button disabled until the format is correct but i'm not sure if that would fall within the scope of this issue

Reply:
actually i went ahead and made this change and dropped the snackbar error, i think this is a lot nicer!